fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -2903,13 +2905,23 @@ private void validateInteraction(String interaction, String resourceType) throws
             if (resourceSpecificInteractions != null) {
                 interactions = resourceSpecificInteractions;
             } else {
-                List<String> defaultInteractions = FHIRConfigHelper.getStringListProperty(defaultInteractionsConfigPath.toString());
-                if (defaultInteractions != null) {
-                    interactions = defaultInteractions;
+                // Check the 'open' property, and if that's false, check if resource was specified
+                if (!FHIRConfigHelper.getBooleanProperty(FHIRConfiguration.PROPERTY_RESOURCES + "/" + FHIRConfiguration.PROPERTY_FIELD_RESOURCES_OPEN, true)) {
+                    PropertyGroup resourceGroup = FHIRConfigHelper.getPropertyGroup(FHIRConfiguration.PROPERTY_RESOURCES + "/" + resourceType);
+                    if (resourceGroup == null) {
+                        resourceValid = false;
+                    }
+                }
+                if (resourceValid) {
+                    // Get the 'Resource' interaction property
+                    List<String> defaultInteractions = FHIRConfigHelper.getStringListProperty(defaultInteractionsConfigPath.toString());
+                    if (defaultInteractions != null) {
+                        interactions = defaultInteractions;
+                    }
                 }
             }
 
-            if (log.isLoggable(Level.FINE)) {
+           if (log.isLoggable(Level.FINE)) {
```

Comment:
inadvertent whitespace deletion.  fix if you want

Reply:
fixed